env/.env.prod_evm

```diff
@@ -0,0 +1,2 @@
+URL="$CF_PAGES_URL"
```

Comment:
For the evm prod env I would recommend using the URL you want the public to see. In this case, It would be eon.horizen.io.

The reason for this is that the places that use this variable, like the metadata will be pointing to the Cloudflare domain with the hashes, ie: askodjfodsaf.cloudflare.com instead of eon.horizen.io